components/omega/doc/design/OmegaV0ShallowWater.md

```diff
@@ -0,0 +1,607 @@
+| $\eta$              | absolute vorticity          | 1/s      | vertex   |   | $\eta=\zeta + f$ |
+| $\kappa_2$          | tracer diffusion            | m^2/s    | cell     |   |                                                              |
+| $\kappa_4$          | biharmonic tracer diffusion | m^4/s    | cell     |   |                                                              |
+| $\nu_2$             | viscosity                   | m^2/s    | edge     |   |                                                              |
+| $\nu_4$             | biharmonic viscosity        | m^4/s    | edge     |   |                                                              |
+| $\phi$              | tracer                      | varies   | cell     |   |                                                              |
+
+Note: Table created with [markdown table generator](https://www.tablesgenerator.com/markdown_tables) and original [google sheet](https://docs.google.com/spreadsheets/d/1rz-QXDiwfemq5NpSR1XsvomI7aSKQ1myTNweCY4afcE/edit#gid=0).
+
+
+
+Table 2. Definition of geometric elements used to build the discrete system.
+| Element | Type | Definition | MPAS mesh name | spherical  |
+|---------------------|-----------------------------|----------|----------|--------------------------------------------------------------|
+| $x_i$ | point | Location of center of primal mesh cells (cell center) | (xCell, yCell, zCell) | (lonCell, latCell) |
+| $x_e$ | point | Location of edge points where velocity is defined | (xEdge, yEdge, zEdge) | (lonEdge, latEdge) |
+| $x_v$ | point | Location of center of dual-mesh cells (Vertex)| (xVertex, yVertex, zVertex) | (lonVertex, latVertex) |
+| $d_e$ | line | segment distance between neighboring $x_i$ locations | dcEdge | |
+| $l_e$ | line | segment distance between neighboring $x_v$ locations | dvEdge | |
+| $P_i$ | cell | A cell on the primal mesh | | |
+| $D_v$ | cell | A cell on the dual-mesh | | |
+
+![Variable positions for MPAS mesh specification.](../_static/variablePosition.png "Variable positions")
+
+Figure 1. Variable positions for MPAS mesh specification.
+
+Table 3. Definition of element groups used to build the discrete system.
+
+| Syntax              | Definition                  | MPAS mesh name | 
+|---------------------|-----------------------------|---|
+| $e\in EC(i)$ |  Set of edges that define the boundary of $P_i$ | edgesOnCell |
+| $e\in EV(v)$ |  Set of edges that define the boundary of $D_v$ | edgesOnVertex |
+| $i\in CE(e)$ |  Two primal mesh cells that share edge $e$ | cellsOnEdge |
+| $i\in CV(v)$ |  Set of primal mesh cells that form the vertices of dual mesh cell $D_v$ | cellsOnVertex |
+| $v\in VE(e)$ |  The two dual-mesh cells that share edge $e$ | verticesOnEdge |
+| $v\in VI(i)$ |  The set of dual-mesh cells that form the vertices of primal mesh cell $P_i$ | verticesOnCell |
```

Comment:
Using the notation in [Ringler 2010](https://www.sciencedirect.com/science/article/pii/S0021999109006780#tbl2), the `i` is used for cell center. I think they chose that because subscript `c` and `e` look the same.

Reply:
@mark-petersen I am not insisting on this change, but I think that this is simply a typo in the table in Ringler 2010. See Eq (26) therein where this variable is called `VC` and Table 2.2 in a later [TRiSK extension paper](https://doi.org/10.1137/110850293).

Reply:
Thanks. You are right, I see that now. It was also correct in the original [Thuburn et al. 2009](http://www.sciencedirect.com/science/article/pii/S0021999109004434).